lib/db/init.ts

```diff
@@ -0,0 +1,41 @@
+import {db} from "../../lib/db";
+import * as fs from "fs";
+
+export function initDB() {
+    const ELW_COUNT: number = 309;
+    const ECS_COUNT: number = 529;
+    const ELW_ID = 1;
+    const ECS_ID = 2;
+
+    // clear the database
+
+    db.prepare("DROP TABLE lockers").run();
+    db.prepare("DROP TABLE buildings").run();
+    db.prepare("DROP TABLE users").run();
+    db.prepare("DROP TABLE registrations").run();
+
+    const schema = fs.readFileSync("schema.sql", "utf8")
+    db.exec(schema);
+
+    // insert the buildings
+
+    const insertBuilding = db.prepare("INSERT INTO buildings (name) VALUES (?)");
+
+    insertBuilding.run("engineering lab wing");
+    insertBuilding.run("engineering computer science");
+
+    // insert the lockers
+
+    const insertLocker = db.prepare("INSERT INTO lockers (building_id, num) VALUES (?,?)");
+
+    let insertCount: number = 0;
+    for (let i = 1; i <= ELW_COUNT; i++) {
+        insertCount += insertLocker.run(ELW_ID, i).changes;
+    }
+
+    for (let i = 1; i <= ECS_COUNT; i++) {
+        insertCount += insertLocker.run(ECS_ID, i).changes;
+    }
```

Comment:
There is a way to execute this in one swift go without needing to have multiple calls to `run` here.
> The first form (with the "VALUES" keyword) creates one or more new rows in an existing table. 
https://www.sqlite.org/lang_insert.html

Also, a little primer on what `prepare` means here (hint, you don't really need it here)
https://en.wikipedia.org/wiki/Prepared_statement
Don't need to implement it but thought I'd mention it here.